hack/conformance.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# This script should be executed through the makefile via `make conformance`.
+
+set -euo pipefail
```

Comment:
why don't we use the x flag?

Reply:
I wanted to make it pretty by showing `echo` logs when running locally. Would you find it more useful if it had `-x`? I don't mind adding it.

Reply:
I get what you mean, but I think in the end these prints can be useful to see what actually happens. so I would vote to add it.

Reply:
Added